lib/exzeitable/html/helpers.ex

```diff
@@ -2,8 +2,20 @@ defmodule Exzeitable.HTML.Helpers do
   @moduledoc "Helpers that are needed across multiple modules in the HTML context"
 
   alias Phoenix.HTML.Tag
+  alias Exzeitable.Params
+
+  @doc "Adds the target to an <a> when there is an id present in the params, for live component pages"
+  @spec tag(any, atom, keyword | Params.t(), keyword) :: {:safe, iolist}
```

Comment:
I added this airity so that for any click `a` tags , if there is an `id` in the params, which indicate that this is a livecomponent, then the target for the events is the livecomponent id, which MUST be added to the DOM id of the livecomponent to work